### PR TITLE
Updated CRDs in operator chart

### PR DESCRIPTION
**Pull request by Amper**

### CODE REVIEW

File: charts/victoria-metrics-operator/templates/crd.yaml [hunk 1]
```diff
@@ -11679,7 +11684,7 @@ spec:
                       acceptEULA:
                         description: AcceptEULA accepts enterprise feature usage,
                           must be set to true. otherwise backupmanager cannot be added
-                          to single/cluster version. https://victoriametrics.com/legal/esa/
+                          to single/cluster version. https://victoriametrics.com/legal/eula/
```
Comment (hagen1778) [inline charts/victoria-metrics-operator/templates/crd.yaml:11687]: Why this change? I believe @tenmozes deliberately changed eula to esa

Comment (Amper) [reply charts/victoria-metrics-operator/templates/crd.yaml:11687]: I just updated the CRD from the release bundle, and the @tenmozes changes aren't released yet. But i can leave them, because it doesn't affect anything. I’ll fix it.

Comment (Amper) [reply charts/victoria-metrics-operator/templates/crd.yaml]: Fixed
